doc/classes/Basis.xml

```diff
@@ -190,6 +243,18 @@
 			<return type="Basis" />
 			<description>
 				Returns the transposed version of the matrix.
+				[codeblock]
+				var a = Basis(
+				    Vector3(1, 2, 3),
+				    Vector3(4, 5, 6),
+				    Vector3(7, 8, 9)
+				)
+
+				var b = a.transposed()
+				print(b.x) # Prints (1, 4, 7).
+				print(b.y) # Prints (2, 5, 8).
+				print(b.z) # Prints (3, 6, 9).
+				[/codeblock]
```

Comment:
Mickeon for the use of basis in orientation, it's typically 1, 0, 0, 0, 1, 0, 0, 0, 1. Like normalized.

Reply:
Like around one, not like 9.

Reply:
Oh this is in the wrong section header.

Reply:
The docs for this & similar really want to express it as working with a 3x3 matrix, no mention of basis. For this case I feel like the starting value is appropriate, since transposing is a broad concept that applies to all matrices, not just the Basis type.